privval/msgs_test.go

```diff
@@ -35,6 +35,10 @@ func exampleVote() *types.Vote {
 		},
 		ValidatorAddress: crypto.AddressHash([]byte("validator_address")),
 		ValidatorIndex:   56789,
+		VoteExtension: types.VoteExtension {
+			AppDataSigned: []byte("app_data_signed"),
```

Comment:
i know in the call last week we discussed changing these names before the merge. I can't remember if we came up with a better name, if not lets do it later

Reply:
We have decided to use `*ToSign` instead of `*Signed`.